### ISSUE TITLE
multi-line addresses from google imports only first line

### ISSUE THREAD

**Issue by ghost** [WhisperSystems/Flock#24]
TLDR; When address in a google contact has multiple lines, only the first line is imported on re-sync.

Example:
123 Main Street
Anycity, NY 10000
becomes just...
123 Main Street

To Reproduce:  
Install flock (v6.0).  
Create new account (Flock server).
Import Google contacts.
At this point addresses seem to have properly imported all lines of multi-line address contacts.  I can hide the google contacts and flock addresses seem accurately to have imported.
Go to another device with no contacts or link to google.
Install flock and connect to same account and sync/download contacts.
When sync is complete, all multi-line addresses have sycn'd with only the first line of the address.
